wallet-new/test/golden/api-layout.txt

```diff
@@ -0,0 +1,48 @@
+/
+├─ addresses/
+│  ├─•
+│  ┆
+│  ├─•
+│  ┆
+│  ┆
+│  └─ <capture>/
+│     └─•
+├─ node-info/
+│  └─•
+├─ node-settings/
+│  └─•
+├─ transactions/
+│  ├─ fees/
+│  │  └─•
+│  ├─•
+│  ┆
+│  └─•
+└─ wallets/
+   ├─•
+   ┆
+   ├─•
+   ┆
+   ┆
+   ├─ <capture>/
+   │  ├─ password/
+   │  │  └─•
+   │  ├─•
+   │  ┆
+   │  ├─•
+   │  ┆
+   │  └─•
+   ┆
+   └─ <capture>/
+      └─ accounts/
+         ├─ <capture>/
+         │  ├─•
+         │  ┆
+         │  └─•
+         ┆
+         ├─•
+         ┆
+         ├─•
+         ┆
+         ┆
+         └─ <capture>/
+            └─•
```

Comment:
this looks a lot less useful than i thought it would be, to be honest?

Reply:
I mean, it is still *useful* but the output could render more useful information.

Reply:
Agreed 😿